app/views/hiring_staff/vacancies/edit.html.haml

```diff
@@ -1,149 +1,169 @@
-        %dd.cya-answer
-          = @vacancy.main_subject
-        %dd.cya-change
-          = link_to t('buttons.change'), edit_school_job_job_specification_path(@vacancy.id, anchor: 'subject')
-      %div
-        %dt.cya-question
-          = t('jobs.salary_range')
-        %dd.cya-answer
-          = @vacancy.salary_range("to")
-        %dd.cya-change
-          = link_to t('buttons.change'), edit_school_job_job_specification_path(@vacancy.id, anchor: 'salary_range')
-      %div
-        %dt.cya-question
-          = t('jobs.working_pattern')
-        %dd.cya-answer
-          = @vacancy.working_pattern.humanize
-        %dd.cya-change
-          = link_to t('buttons.change'), edit_school_job_job_specification_path(@vacancy.id, anchor: 'working_pattern')
-      %div
-        %dt.cya-question
-          = t('jobs.flexible_working')
-        %dd.cya-answer
-          = @vacancy.flexible_working
-        %dd.cya-change
-          = link_to t('buttons.change'), edit_school_job_job_specification_path(@vacancy.id, anchor: 'flexible_working')
-      %div
-        %dt.cya-question
-          = t('jobs.benefits')
-        %dd.cya-answer
-          = @vacancy.benefits
-        %dd.cya-change
-          = link_to t('buttons.change'), edit_school_job_job_specification_path(@vacancy.id, anchor: 'benefits')
-      %div
-        %dt.cya-question
-          = t('jobs.pay_scale')
-        %dd.cya-answer
-          =@vacancy.pay_scale_range
-        %dd.cya-change
-          = link_to t('buttons.change'), edit_school_job_job_specification_path(@vacancy.id, anchor: 'pay_scale_range')
-      %div
-        %dt.cya-question
-          = t('jobs.weekly_hours')
-        %dd.cya-answer
-          = @vacancy.weekly_hours
-        %dd.cya-change
-          = link_to t('buttons.change'), edit_school_job_job_specification_path(@vacancy.id, anchor: 'weekly_hours')
-      %div
-        %dt.cya-question
-          = t('jobs.leadership_level')
-        %dd.cya-answer
-          - if @vacancy.leadership
-            = @vacancy.leadership.title
-        %dd.cya-change
-          = link_to t('buttons.change'), edit_school_job_job_specification_path(@vacancy.id, anchor: 'leadership')
-      %div
-        %dt.cya-question
-          = t('jobs.starts_on')
-        %dd.cya-answer
-          = format_date @vacancy.starts_on
-        %dd.cya-change
-          = link_to t('buttons.change'), edit_school_job_job_specification_path(@vacancy.id, anchor: 'starts_on')
-      %div
-        %dt.cya-question
-          = t('jobs.ends_on')
-        %dd.cya-answer
-          = format_date @vacancy.ends_on
-        %dd.cya-change
-          = link_to t('buttons.change'), edit_school_job_job_specification_path(@vacancy.id, anchor: 'ends_on')
+    %table.check-your-answers.cya-questions-short
+      %thead
+        %tr
+          %th{:colspan => "2"}
+            %h2.heading-medium
+              = link_to t('jobs.job_specification'), edit_school_job_job_specification_path(@vacancy.id)
```

Comment:
@gazaston We should add a `Change` link or button on this line with a hidden label for screen readers clarifying that this takes you to where you can amend the specific section. Linking can be confusing both for screen readers but also for normal users. I would have expected this to be an anchored link that brings that section to the top of the page in this instance.

Reply:
Really? I thought the point of us adding these links was that they were purely links to that form page, to answer the issue of letting users *not* link to an anchored part of the page.